algosdk/atomic_transaction_composer.py

```diff
@@ -68,6 +78,13 @@ def populate_foreign_array(
     return offset + len(foreign_array) - 1
 
 
+SignedTransaction = Union[
```

Comment:
@jdtzmn @algochoi Based on usage, I'm skeptical the type alias here is the intended type definition.  Can you have a look at https://github.com/jdtzmn/py-algorand-sdk/pull/2/ to see if you agree with the proposed change?

Reply:
@algochoi Oh - I think I misinterpreted `stxns = signer.sign_transactions(txns, indexes)`.  I've run out of time at the moment, but want to note that it seems like https://github.com/jdtzmn/py-algorand-sdk/pull/2 does _not_ make sense.  At least in its current form.  Need to reconsider more deeply.

Reply:
@algochoi @jdtzmn A few thoughts:
* I closed https://github.com/jdtzmn/py-algorand-sdk/pull/2.  I think it's incorrect and I think this PR is correct.
* I have a passing concern with naming the alias `SignedTransaction` for fear of colliding with `transaction.SignedTransaction`.  
  * Admittedly, I don't see a great alternative unless folks feel `IsSigned` or similar is _reasonable_.
  * If there's no obvious alternative, then let's keep as is.
* I opened another attempt to improve return type annotations following Andrew's suggestion:  https://github.com/jdtzmn/py-algorand-sdk/pull/3.  Can you have a look to see if it's worth pulling in?

Reply:
Child PR looks good, I don't have a good alternative to the naming either.

Reply:
I have merged the child PR - thank you for the changes. I also renamed `SignedTransaction` to `GenericSignedTransaction` to attempt to differentiate the two. Resolving this conversation for now.

Reply:
@jdtzmn Thanks for the rename - it ameliorates my concern that a builder will errantly use the unintended `SignedTransaction` type.